pages/user guides/public_urls.md

```diff
@@ -0,0 +1,52 @@
+---
+toc: false
+title: Public URL Discovery
+keywords: urls, , discover, 
+last_updated: January 19, 2021
+summary: "Discover your organizations public URLs"
+sidebar: main_sidebar
+folder: userguides
+permalink: public-urls.html
+---
+
+# Discover your organizations public URLs
+We will start with a single domain that you know of for your organization, and in the end, discover
+and capture a new set of other domains and URLs that are publicly available for your organization. 
+The entire process should only take a couple of minutes and does not require any private network access.
+
+1. You can start with a top level domain that your company owns. For example, canadiantire.ca.
+
+Analyze the domain with Tidal Tools.[link to other guide] - `tidal analyze web canadiantire.ca`
```

Comment:
QQ, is the plan to add the link later on?

Reply:
Oh no, plan is to add it now. I will update this so it is correct. Thanks!